<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="4">
      <layout class="QVBoxLayout" name="graphLayout"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X-axis min</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxXMin">
       <property name="minimum">
        <number>-1000000</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>-10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X-axis max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxXMax">
       <property name="minimum">
        <number>-1000000</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinBoxYMin">
       <property name="minimum">
        <number>-1000000</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>-10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y-axis max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinBoxYMax">
       <property name="minimum">
        <number>-1000000</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Y-axis min</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QPushButton" name="plotButton">
       <property name="text">
        <string>Построить!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
